### PR TITLE
REF: Use walletID instead of secret

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/bw2qyg4f5zrv3zvj3yn443wy6c?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/walletid-1623965850.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/d0a8phw1b0rff6u5119dxrh164?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/walletid-1624033541.apk)

File: screen/wallets/marketplace.js [hunk 1]
```diff
@@ -78,8 +80,10 @@ Marketplace.propTypes = {
   }),
 };
 
+Marketplace.contextType = BlueStorageContext;
+
 Marketplace.navigationOptions = navigationStyle({
   closeButton: true,
-  title: 'Marketplace',
+  title: loc.wallets.list_marketplace,
```
Comment (Overtorment) [inline screen/wallets/marketplace.js:87]: good find

**Review by Overtorment — APPROVED** [review]
lgtm

**Comment by Overtorment** [comment]
@ncoelho wake the fuck up samurai, we have prs to merge

**Review by ncoelho — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**